tracks/rust/exercises/sublist/mentoring.md

```diff
@@ -0,0 +1,45 @@
+### Concepts
+
+- enum
+- generic over type
+
+### Reasonable solutions
+
+A reasonable solution should do the following:
+
+- break down the unordered size problem into two subproblems, checking if a
+  list is smaller than another, and checking if a smaller list is a sublist of
+  a larger list
+- Utilize Eq for slices
+- not reimplement the functionality of the `windows` method
+
+### Examples
+
+```rust
+#[derive(Debug, PartialEq)]
+pub enum Comparison {
+    Equal,
+    Sublist,
+    Superlist,
+    Unequal,
+}
+
+pub fn sublist<T: PartialEq>(_first_list: &[T], _second_list: &[T]) -> Comparison {
```

Comment:
Shouldn't these be named without the underscore, as the arguments are used in the functions body?